packages/block-library/src/query-title/block.json

```
@@ -26,7 +26,15 @@
 		"typography": {
 			"fontSize": true,
 			"lineHeight": true,
-			"__experimentalFontFamily": true
+			"__experimentalFontStyle": true,
+			"__experimentalFontWeight": true,
+			"__experimentalLetterSpacing": true,
+			"__experimentalTextTransform": true,
+			"__experimentalDefaultControls": {
+				"fontSize": true,
+				"fontAppearance": true,
+				"__experimentalTextTransform": true
```

This also needs a tweak.

```suggestion
				"textTransform": true
```